navbar-component.html

```diff
@@ -4,33 +4,36 @@
       <div class="navbar-header">
         <div br-slot="br-navbar-header">
           <div br-slot="br-navbar-header-inner"></div>
-          <button ng-if="$ctrl.getDisplayedMenus().length > 0" type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#br-navbar-collapse" aria-expanded="false">
+          <button ng-if="$ctrl.getDisplayedMenus().length > 0 && $ctrl.shouldDisplay('menu')"
+            type="button" class="navbar-toggle collapsed" aria-expanded="false"
+            ng-click="$ctrl.isNavCollapsed = !$ctrl.isNavCollapsed">
             <span class="sr-only">Toggle navigation</span>
             <span class="icon-bar"></span>
             <span class="icon-bar"></span>
             <span class="icon-bar"></span>
           </button>
-          <a class="navbar-brand" href="/">
+          <a ng-if="$ctrl.shouldDisplay('brand')" class="navbar-brand" href="{{$ctrl.siteUrl()}}">
```

Comment:
Use ng-href instead of just href. Prevents people from clicking the link before the expression is resolved.